mw-webapp/src/logic/waysTable/columns.tsx

```diff
@@ -68,7 +78,11 @@ export const columns = [
      */
     cell: ({row}) => {
       return (
-        row.original.currentMentors.map((currentMentor) => renderCellValue(currentMentor.name))
+        row.original.currentMentors.map((currentMentor) => (
+          <div key={currentMentor.uuid}>
```

Comment:
if you are using div element only for adding key attribute you can use React.Fragment

Reply:
Done.